proto/config/netcmn.proto

```diff
@@ -83,12 +83,25 @@ message ipspec {
   string subnet = 3;
   string gateway = 5;
   string domain = 6;
+  // ntp and more_ntp are used to specify several NTP servers
   string ntp = 7;
+  repeated string more_ntp = 10;
   repeated string dns = 8;
 
   // for IPAM management when dhcp is turned on.
   // If none provided, system will default pool.
   ipRange dhcpRange = 9;
+
+  DhcpOptionsIgnore dhcp_options_ignore = 11;
```

Comment:
I am not sure if we should put meaning in it if it is `nil` (because I don't know if other programming languages support this). Therefore if it is `nil` it should be handled as if the default value is set, i.e. `false`.
Therefore if `dhcp_options_ignore` is not defined, "use NTP from both DHCP and the static config".

If you want to use NTP from DHCP only, set `ntp` and `more_ntp` to empty strings.

Do we do this somewhere else? If yes, then it is perhaps better to do it the way you suggest.

Reply:
I see. No, I'm not aware of any place where we would check if struct (proto message) is set or unset.
Your approach makes sense to me now. I would maybe just suggest to add a comment (next to `dhcp_options_ignore`  ?) that even if `DhcpType` is `Client`, EVE still accepts static IP config and will merge it with the DHCP-provided one.

Reply:
In the protobuf wire encoding the fields with default values are not sent. Thus e.g., a boolean which is false, an integer of value zero, is not sent. As a result the receiver can not tell whether the sender explicitly set it to false or zero in those examples. I imagine this applies to a struct/embedded message as well; if all fields are zero it might not be sent at all.
I don't know if this is useful.